fake-files/output-expected/linnarsson/linnarsson.images.json

```diff
@@ -1,10 +1,10 @@
 {
   "polyT": {
     "sample": 1,
-    "tileSource": "https://s3.amazonaws.com/vitessce-data/0.0.20/master_release/linnarsson/linnarsson.tiles/linnarsson.images.polyT/polyT.dzi"
+    "tileSource": "https://s3.amazonaws.com/vitessce-data/0.0.20/master_release/linnarsson/linnarsson.images.zarr/pyramid/"
```

Comment:
Can you update the directory we're pushing to?

Reply:
I would do this, but I'm not sure what that should be in google cloud! 

I pushed the data to `https://vitessce-data.storage.googleapis.com/linnarsson.images.zarr/pyramid` to test with the current version of viv, but I think we should version this end-point.

Reply:
Add other necessary metaddata (zoom level)

Reply:
For URL: https://github.com/hubmapconsortium/vitessce-data/pull/73/files